src/core/background/scrobbler/lastfm.js

```diff
@@ -79,9 +79,11 @@ define((require) => {
 			let trackUrl = $doc.find('track > url').text();
 			let albumUrl = $doc.find('album > url').text();
 
+			let userPlayCount = $doc.find('userplaycount').text();
```

Comment:
Would it make sense to turn this into an int and treat it as such through the pipeline? 


```suggestion
			let userPlayCount = parseInt($doc.find('userplaycount').text());
```

Reply:
Yes, absolutely! There was a voice in my head that told me to go the integer way. But I guess the string voice somehow won over and made me lazy 😜. Thanks for bringing it up. My latest commit fixes this.